python/docs/conf.py

```diff
@@ -0,0 +1,62 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'interpret-community'
+copyright = '2019, Microsoft'
+author = 'Microsoft'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0.5'

```

Comment:
is this our version, or sphinx version?  can we automatically fetch it from somewhere, otherwise we will have to update it on every version change?

Reply:
I'll ask Richard about it today. I think that if we incorporate doc building into our release pipeline, we should be able to inject this. For now, the build is happening manually so it needs to be updated manually.